eng/common/generate-sbom-prep.ps1

```diff
@@ -0,0 +1,19 @@
+Param(
+    [Parameter(Mandatory=$true)][string] $ManifestDirPath    # Manifest directory where sbom will be placed
+)
+
+Write-Host "Creating dir $ManifestDirPath"
+# create directory for sbom manifest to be placed
+if (!(Test-Path -path $ManifestDirPath))
+{
+  New-Item -ItemType Directory -path $ManifestDirPath
+  Write-Host "Successfully created directory $ManifestDirPath"
+}
+else{
+  Write-PipelineTelemetryError -category 'Build'  "Unable to create sbom folder."
```

Comment:
>Write-PipelineTelemetryError -category 'Build'  "Unable to create sbom folder." [](http://example.com/codeflow?start=2&length=79)

Is it possible the folder already created?

Reply:
You can share this feedback here: https://github.com/dotnet/arcade/pull/8479